store/sqlstore/thread_store.go

```diff
@@ -52,7 +52,7 @@ func threadToSlice(thread *model.Thread) []interface{} {
 		thread.ChannelId,
 		thread.LastReplyAt,
 		thread.ReplyCount,
-		thread.Participants,
+		model.ArrayToJson(thread.Participants),
```

Comment:
`Thread.Save` was broken because `Participants` (model.StringArray) wouldn't serialize properly.
We didn't actually use `Thread.Save` anywhere so we never noticed.
We use the `Exec` method to hack around gorp not supporting inserting multiple rows in a single transaction,
but `Exec` method doesn't call the `TypeConverter`, so here we explicitly convert it to the right format.